spec/refresher_spec.rb

```diff
@@ -5,70 +5,71 @@
   context "guess the collection" do
 
     it "Yes, it is a Object, but which class" do
-      expect([].is_a? _fill_in_object_).to be true
-      expect(Array.new.instance_of? _FILL_ME_IN_).to be true
+      expect([].is_a? Array).to be true
+      expect(Array.new.instance_of? Array).to be true
     end
 
     it "Yes, it is a Object, but which class" do
-      expect({}.is_a? _fill_in_object_).to be true
-      expect(Hash.new.instance_of? _FILL_ME_IN_).to be true
+      expect({}.is_a? Hash).to be true
+      expect(Hash.new.instance_of? Hash).to be true
     end
   end
 
   context "guess the type" do
     it "Yes, it is a Object, but which" do
-      expect(:class.is_a? _fill_in_object_).to be true
+      expect(:class.is_a? Symbol).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect("module".is_a? _fill_in_object_).to be true
+      expect("module".is_a? String).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect(1.is_a? _fill_in_object_).to be true
+      expect(1.is_a? Integer).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect(1.5.is_a? _fill_in_object_).to be true
+      expect(1.5.is_a? Float).to be true
     end
   end
 
   context "the following are Array methods" do
     it "adds a thing to the end of the method" do
-      expect([]._fill_in_method_here_(1)).to eq [1]
+      expect([].push(1)).to eq [1]
     end
 
     it "removes an item from the end of an array" do
-      expect([1]._fill_in_method_here_).to eq 1
+      expect([1].pop).to eq 1
     end
 
     it "adds an item to the front of an array" do
-      expect([1]._fill_in_method_here_("banana")).to eq ["banana", 1]
+      expect([1].unshift("banana")).to eq ["banana", 1]
     end
 
     it "removes an item from the front of an array" do
-      expect([1, "banana"]._fill_in_method_here_).to eq ["banana"]
+      expect([1, "banana"].drop(1)).to eq ["banana"]
```

Comment:
It's funny that you mention that (shift); I tried that initially but it wouldn't accept it and I'm not sure why.

---- On Mon, 16 Mar 2015 04:20:52 +0100 jaybobo &lt;notifications@github.com&gt; wrote ---- 

In spec/refresher_spec.rb:
 &gt; end &gt; &gt; it "removes an item from the front of an array" do &gt; - expect([1, "banana"]._fill_in_method_here_).to eq ["banana"] &gt; + expect([1, "banana"].drop(1)).to eq ["banana"]  nice. .shift would also work here.
 —
Reply to this email directly or view it on GitHub.